config/samples/flows_v1beta1_flowcollector.yaml

```diff
@@ -94,6 +95,7 @@ spec:
     maxBackoff: 5s
     maxRetries: 2
   consolePlugin:
+    enable: true
```

Comment:
I feel it would be usefull to be able to disable console plugin even using Loki.
WDYT ?

Reply:
why not
a downside is adding more settings always makes it feels more complicated to the user - I think we tend to have a bunch of never-used settings that over-complicate the experience
but a "enable" flag looks simple enough to understand so that's fine with me :-)